tests/rna_io/CMakeLists.txt

```diff
@@ -15,7 +15,9 @@ message (STATUS "Configuring tests/rna_io")
 # ----------------------------------------------------------------------------
 
 # Search SeqAn and select dependencies.
-find_package (SeqAn REQUIRED)
+if (NOT "${SEQAN_BUILD_SYSTEM}" STREQUAL "DEVELOP")
```

Comment:
why is this only for these two tests necessary?
And when are tests not build in `DEVELOP` mode?

Reply:
> why is this only for these two tests necessary?

It's necessary for all tests, but these two deviated from the rest by doing it differently.

> And when are tests not build in DEVELOP mode?

Normally not at all, but the important thing is that it is DEVELOP mode that makes the dependencies be detected first. I guess we could remove these checks from the tests altogether, but that would be a different PR. This PR brings the two tests in line with the rest.